static/main.css

```diff
@@ -732,3 +732,194 @@ footer a:hover {
   border: 2px solid rgba(0, 0, 0, 0.2);
   border-radius: 5px;
 }
+
+
+
+
+
+
+/*contact us styling*/
+input[type=text], select, textarea {
+  width: 100%;
+  padding: 12px;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+  margin-top: 6px;
+  margin-bottom: 16px;
+  resize: vertical;
+  
+}
+
+input[type=submit] {
+  background-color: #ffffff;
+  color: #021928;
+  padding: 12px 20px;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+  font-weight: 900;
+  font-size: large;
+}
+
+input[type=submit]:hover {
+  background-color: #021928;
+}
+
+.container {
+  border-radius: 5px;
+  background-color: #021928;
+  padding: 20px;
+}
+.contact{
+font-style: normal;
+font-size: 50px;
+font-weight: 900;
+}
+
+/*about us */
+html{
+  scroll-behavior: smooth;
+}
+body{
+  background: #021928;
```

Comment:
Det ser ut att du har ändrad body background på alla sidor. Det ser inte så bra ut på index och challenges. Kan du kanske skapa en class eller id på about us and story och då använda den i css?

Reply:
Ska göra det !

Reply:
Gjort!